### DISCUSSION TITLE
Using ref in Controller

### DISCUSSION THREAD

**Discussion by tempoeric** [react-hook-form/react-hook-form#6773] [Q&A] [answered]
Hi i just wanna ask how to use ref inside a controller
![image](https://user-images.githubusercontent.com/42777871/136901358-c9120e5d-86b7-439d-bb70-e81a42ea63da.png)

**Comment by tempoeric** [comment]
@bluebill1049  i hope you can help me

**Comment by bluebill1049** [comment, marked as answer]
https://react-hook-form.com/api/usecontroller/controller 

![Screen Shot 2021-10-12 at 6 57 29 pm](https://user-images.githubusercontent.com/10513364/136915699-beb3678e-f042-4d36-aad4-728e4370f911.png)

**Reply by bluebill1049** [reply]
avoid double `register`
